### PR TITLE
HDFS-15044. [Dynamometer] Show the line of audit log when parsing it unsuccessfully.

### PR DESCRIPTION
**Pull request by tasanuma**
## NOTICE

Please create an issue in ASF JIRA before opening a pull request,
and you need to set the title of the pull request which starts with
the corresponding JIRA issue number. (e.g. HADOOP-XXXXX. Fix a typo in YYY.)
For more details, please see https://cwiki.apache.org/confluence/display/HADOOP/How+To+Contribute

### CODE REVIEW

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m  0s |  Docker mode activated.  |
| -1 :x: |  docker  |   0m 10s |  Docker failed to build yetus/hadoop:104ccca9169.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hadoop/pull/1749 |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1749/1/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: hadoop-tools/hadoop-dynamometer/hadoop-dynamometer-workload/src/main/java/org/apache/hadoop/tools/dynamometer/workloadgenerator/audit/AuditLogDirectParser.java [hunk 1]
```diff
@@ -125,8 +125,9 @@ public AuditReplayCommand parse(Text inputLine,
       relativeTimestamp = dateFormat.parse(m.group("timestamp")).getTime()
           - startTimestamp;
     } catch (ParseException p) {
-      throw new IOException("Exception while parsing timestamp from audit log",
-          p);
+      throw new IOException(
+          "Exception while parsing timestamp from audit log line: `"
```
Comment (xkrogen) [inline hadoop-tools/hadoop-dynamometer/hadoop-dynamometer-workload/src/main/java/org/apache/hadoop/tools/dynamometer/workloadgenerator/audit/AuditLogDirectParser.java]: You have an extra backtick here. There should either be no backticks or one before and one after the `inputLine`.

**Comment by tasanuma** [comment]
Thanks for your review, @xkrogen. I'd like to add backticks. Updated the PR.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m  0s |  Docker mode activated.  |
| -1 :x: |  docker  |   0m 18s |  Docker failed to build yetus/hadoop:104ccca9169.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hadoop/pull/1749 |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1749/2/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by xkrogen** [comment]
Looks like Yetus is failing due to HADOOP-16754. It seems that it is about to be fixed, I guess we can wait for that to make sure we get a Yetus run.

**Comment by aajisaka** [comment]
retest this please

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  6s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  18m 15s |  trunk passed  |
| +1 :green_heart: |  compile  |   0m 26s |  trunk passed  |
| +1 :green_heart: |  checkstyle  |   0m 20s |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   0m 27s |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  13m 23s |  branch has no errors when building and testing our client artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 23s |  trunk passed  |
| +0 :ok: |  spotbugs  |   0m 37s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   0m 34s |  trunk passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 23s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 18s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 18s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 13s |  the patch passed  |
| +1 :green_heart: |  mvnsite  |   0m 20s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedclient  |  13m  7s |  patch has no errors when building and testing our client artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 20s |  the patch passed  |
| +1 :green_heart: |  findbugs  |   0m 39s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 54s |  hadoop-dynamometer-workload in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 32s |  The patch does not generate ASF License warnings.  |
|  |   |  53m 24s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1749/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1749 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux c3102b70313a 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 93bb368 |
| Default Java | 1.8.0_222 |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1749/4/testReport/ |
| Max. process+thread count | 412 (vs. ulimit of 5500) |
| modules | C: hadoop-tools/hadoop-dynamometer/hadoop-dynamometer-workload U: hadoop-tools/hadoop-dynamometer/hadoop-dynamometer-workload |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1749/4/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**